scripts/test-setup.sh

```diff
@@ -8,6 +8,15 @@ az group create \
 --location ${TEST_LOCATION} \
 -o none
 
+echo "create managed identity"
```

Comment:
moved that higher up to make sure the identity is there when creating the roleassignment (azure graph race)